package.json

```diff
@@ -62,7 +51,6 @@
   },
   "lint-staged": {
```

Comment:
lint-staged composes?

Reply:
you're right, you've fixed it at https://github.com/status-im/status-web/pull/417/commits/1f092b0e0b37b8c11671a2166e5ff7aab2d820de